JavaHackerrank/Optimal_Network_Routing.java

```diff
@@ -0,0 +1,114 @@
+import java.io.*;
```

Comment:
Problem Link : https://www.hackerrank.com/contests/hack-the-interview-iv-apac/challenges/optimal-path-1/problem